centreon/www/include/views/componentTemplates/DB-Func.php

```diff
@@ -49,12 +49,12 @@ function DsHsrTestExistence($name = null)
 
     $query = 'SELECT compo_id FROM giv_components_template WHERE ds_name = :ds_name';
 
-    if (!empty($formValues['host_id'])) {
-        if (preg_match('/([0-9]+)-([0-9]+)/', $formValues['host_id'], $matches)) {
+    if (!empty($formValues['host_service_id'])) {
+        if (preg_match('/([0-9]+)-([0-9]+)/', $formValues['host_service_id'], $matches)) {
             $formValues['host_id'] = (int) $matches[1];
             $formValues['service_id'] = (int) $matches[2];
         } else {
-            throw new \InvalidArgumentException('host_id must be a combination of integers');
+            throw new \InvalidArgumentException('host_service_id must be a combination of integers');
         }
     }
```

Comment:
same here, directly use `list($formValues['host_id'], $formValues['service_id']) = parseHostIdPostParameter($formValues['host_service_id']);` ?

Reply:
Same (sure that's the all point).